tag_dedupe/dedupe.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python
+
+import sys
+import subprocess
+import os
+import argparse
+from sets import Set
+
+AUTH_FILE_PATH_LOCAL = "/auth.json"
+
+def dedupe(input_tag, existing_tags):
+	for tag in existing_tags:
+		if input_tag == tag:
```

Comment:
this is exactly what I meant to do when I changed this to use a set...just forgot to do it haha. done